files/layzr.js/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "layzr.js",
+  "repo": "callmecavs/layzr.js",
+  "files": {
+    "basePath": "dist/"
```

Comment:
can you add an `include: *.min.*` parameter(with correct formatting)? To make sure we host only the minified files.